.github/ci/files/public/index_test.php

```diff
@@ -25,21 +25,25 @@
 define('APP_ENV', 'test');
 
 Bootstrap::setProjectRoot();
-Bootstrap::bootstrap();
 
 $request = Request::createFromGlobals();
 
+return function (Request $request, array $context) {
 // set current request as property on tool as there's no
 // request stack available yet
-Tool::setCurrentRequest($request);
+    Tool::setCurrentRequest($request);
 
-/** @var \Pimcore\Kernel $kernel */
-$kernel = Bootstrap::kernel();
+    Bootstrap::bootstrap();
+
+    /** @var \Pimcore\Kernel $kernel */
+    $kernel = Bootstrap::kernel();
 
 // reset current request - will be read from request stack from now on
-Tool::setCurrentRequest(null);
+    Tool::setCurrentRequest(null);
 
-$response = $kernel->handle($request);
-$response->send();
+    $response = $kernel->handle($request);
+    $response->send();
 
-$kernel->terminate($request, $response);
+    $kernel->terminate($request, $response);
+    return $kernel;
```

Comment:
unsure about this, if it's terminated 
https://github.com/pimcore/skeleton/blob/0533f1043074e3adbd781cd4e5f11750459daba3/public/index.php#L35